third_party/jpeg/jpeg.BUILD

```diff
@@ -5,6 +5,8 @@ licenses(["notice"])  # custom notice-style license, see LICENSE.md
 
 exports_files(["LICENSE.md"])
 
+load("@//third_party:common.bzl", "expand_header_template")
```

Comment:
Probably not worthwhile. We both know what needs to be done. We'll do it as soon as we can.